extruct/jsonld.py

```diff
@@ -28,14 +28,23 @@ def extract_items(self, document, base_url=None):
             if items for item in items if item
         ]
 
+    def _may_be_get_json(self, script):
+        try:
+            return json.loads(script, strict=False)
+        except Exception:
+            return False
```

Comment:
minor, but to me it would make more sense to return ``None`` here, what do you think? Both ``None`` and ``False`` can be valid results of JSON parsing, but ``None`` looks like a more "neutral" value to use for error case.

Reply:
fixed.